src/renderer/components/+cluster/cluster-metrics.module.scss

```diff
@@ -17,7 +17,6 @@
   }
 
   .empty {
-    margin-top: -45px;
```

Comment:
This change (and the one below) is a required part of the fix. With this here it was covering the radio buttons and preventing them from being clicked.